src/Raven.Server/Json/BlittableJsonTextWriterExtensions.cs

```diff
@@ -1029,6 +1029,44 @@ private static void WriteMetadata(BlittableJsonTextWriter writer, Document docum
             writer.WriteEndObject();
         }
 
+        public static void WriteServerStoreObject(this BlittableJsonTextWriter writer, JsonOperationContext context, BlittableJsonReaderObject data, long etag)
```

Comment:
Haven't found a function that writes the @metadata/@etag properties except for the one written specifically for a Document class...